src/utils/util.ts

```diff
@@ -0,0 +1,4 @@
+import fetch from 'node-fetch';
+
+module.exports.makeOption = (name) => ({text: name, value: name});
+module.exports.makeOptions = (names) => names.map((name) => ({text: name, value: name}));
```

Comment:
`text` will change to `label`

```suggestion
module.exports.makeOption = (name) => ({label: name, value: name});
module.exports.makeOptions = (names) => names.map((name) => ({label: name, value: name}));
```

Reply:
done.